.prout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "checkpoints": {
+    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
```

Comment:
Hi,

Thanks for the proactive response.  I was unsure which was the correct way.  However i did try to run the deploy to heroic and it deployed however trying to load it gets an oops error.  I have not yet investigated why this is but if it helps this is the heroic error:

2015-10-21T10:15:15.046230+00:00 app[web.1]: [error] application -
2015-10-21T10:15:15.046235+00:00 app[web.1]: ! @6np0c6eje - Internal server error, for (GET) [/] ->
2015-10-21T10:15:15.046238+00:00 app[web.1]: 1) Error in custom provider, @6np0c6eg3: Configuration error
2015-10-21T10:15:15.046236+00:00 app[web.1]:
2015-10-21T10:15:15.046244+00:00 app[web.1]:   while locating play.api.libs.Crypto
2015-10-21T10:15:15.046237+00:00 app[web.1]:
2015-10-21T10:15:15.046241+00:00 app[web.1]:     for parameter 0 at play.api.libs.Crypto.<init>(Crypto.scala:282)
2015-10-21T10:15:15.046237+00:00 app[web.1]: play.api.UnexpectedException: Unexpected exception[ProvisionException: Unable to provision, see the following errors:
2015-10-21T10:15:15.046239+00:00 app[web.1]:   while locating play.api.libs.CryptoConfigParser
2015-10-21T10:15:15.046240+00:00 app[web.1]:   while locating play.api.libs.CryptoConfig
2015-10-21T10:15:15.046242+00:00 app[web.1]:   at play.api.inject.BuiltinModule.bindings(BuiltinModule.scala:48):
2015-10-21T10:15:15.046245+00:00 app[web.1]:
2015-10-21T10:15:15.046246+00:00 app[web.1]: 1 error]
2015-10-21T10:15:15.046244+00:00 app[web.1]: Binding(class play.api.libs.Crypto to self) (via modules: com.google.inject.util.Modules$OverrideModule -> play.api.inject.guice.GuiceableModuleConversions$$anon$1)
2015-10-21T10:15:15.046252+00:00 app[web.1]: 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:269) [com.typesafe.play.play-netty-server_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046247+00:00 app[web.1]: 	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:261) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046248+00:00 app[web.1]: 	at play.api.GlobalSettings$class.onError(GlobalSettings.scala:179) [com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046248+00:00 app[web.1]: 	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:191) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046249+00:00 app[web.1]: 	at play.api.mvc.WithFilters.onError(Filters.scala:93) [com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046259+00:00 app[web.1]:   while locating play.api.libs.CryptoConfig
2015-10-21T10:15:15.046251+00:00 app[web.1]: 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:273) [com.typesafe.play.play-netty-server_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046258+00:00 app[web.1]:   while locating play.api.libs.CryptoConfigParser
2015-10-21T10:15:15.046260+00:00 app[web.1]:   at play.api.inject.BuiltinModule.bindings(BuiltinModule.scala:48):
2015-10-21T10:15:15.046254+00:00 app[web.1]: 	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [org.scala-lang.scala-library-2.11.7.jar:na]
2015-10-21T10:15:15.046250+00:00 app[web.1]: 	at play.api.http.GlobalSettingsHttpErrorHandler.onServerError(HttpErrorHandler.scala:94) [com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046253+00:00 app[web.1]: 	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [org.scala-lang.scala-library-2.11.7.jar:na]
2015-10-21T10:15:15.046256+00:00 app[web.1]:
2015-10-21T10:15:15.046257+00:00 app[web.1]: 1) Error in custom provider, @6np0c6eg3: Configuration error
2015-10-21T10:15:15.046259+00:00 app[web.1]:     for parameter 0 at play.api.libs.Crypto.<init>(Crypto.scala:282)
2015-10-21T10:15:15.046262+00:00 app[web.1]:   while locating play.api.libs.Crypto
2015-10-21T10:15:15.046255+00:00 app[web.1]: 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [org.scala-lang.scala-library-2.11.7.jar:na]
2015-10-21T10:15:15.046256+00:00 app[web.1]: Caused by: com.google.inject.ProvisionException: Unable to provision, see the following errors:
2015-10-21T10:15:15.046261+00:00 app[web.1]: Binding(class play.api.libs.Crypto to self) (via modules: com.google.inject.util.Modules$OverrideModule -> play.api.inject.guice.GuiceableModuleConversions$$anon$1)
2015-10-21T10:15:15.046263+00:00 app[web.1]: 1 error
2015-10-21T10:15:15.046264+00:00 app[web.1]: 	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1051) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046276+00:00 app[web.1]: 	at scala.Option.fold(Option.scala:158) ~[org.scala-lang.scala-library-2.11.7.jar:na]
2015-10-21T10:15:15.046267+00:00 app[web.1]: 	at play.api.Application$$anonfun$instanceCache$1.apply(Application.scala:234) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046275+00:00 app[web.1]: 	at play.utils.InlineCache.apply(InlineCache.scala:55) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046286+00:00 app[web.1]: 	at com.google.inject.internal.BoundProviderFactory.provision(BoundProviderFactory.java:72) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046266+00:00 app[web.1]: 	at play.api.inject.guice.GuiceInjector.instanceOf(GuiceInjectorBuilder.scala:316) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046275+00:00 app[web.1]: 	at play.utils.InlineCache.fresh(InlineCache.scala:69) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046265+00:00 app[web.1]: 	at play.api.inject.guice.GuiceInjector.instanceOf(GuiceInjectorBuilder.scala:321) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046264+00:00 app[web.1]: 	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1025) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046277+00:00 app[web.1]: 	at play.api.libs.Crypto$.crypto(Crypto.scala:43) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046278+00:00 app[web.1]: 	at play.api.libs.CryptoConfigParser.get$lzycompute(Crypto.scala:236) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046289+00:00 app[web.1]: 	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:62) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046262+00:00 app[web.1]:
2015-10-21T10:15:15.046278+00:00 app[web.1]: Caused by: play.api.PlayException: Configuration error[Application secret not set]
2015-10-21T10:15:15.046285+00:00 app[web.1]: 	at com.google.inject.internal.ProviderInternalFactory.provision(ProviderInternalFactory.java:81) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046287+00:00 app[web.1]: 	at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:62) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046267+00:00 app[web.1]: 	at play.api.Application$$anonfun$instanceCache$1.apply(Application.scala:234) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046279+00:00 app[web.1]: 	at play.api.libs.CryptoConfigParser.get(Crypto.scala:203) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046284+00:00 app[web.1]: 	at play.api.libs.CryptoConfigParser.get(Crypto.scala:197) ~[com.typesafe.play.play_2.11-2.4.3.jar:2.4.3]
2015-10-21T10:15:15.046286+00:00 app[web.1]: 	at com.google.inject.internal.ProviderInternalFactory.circularGet(ProviderInternalFactory.java:61) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046288+00:00 app[web.1]: 	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:38) ~[com.google.inject.guice-4.0.jar:na]
2015-10-21T10:15:15.046290+00:00 app[web.1]: 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:104) ~[com.google.inject.guice-4.0.jar:na]

-- 
graham hadgraft
Sent with Airmail

On 21 October 2015 at 11:11:53, Roberto Tyley (notifications@github.com) wrote:

In .prout.json:

> @@ -0,0 +1,5 @@
> +{
> +  "checkpoints": {
> +    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
Hi @bitscraps - you actually have a JSON formatting error here, the trailing comma on this line makes this invalid JSON.

Something I haven't been very clear about is whether people should use their own copy of Prout, or use the instance we have set up for the Guardian at https://prout-bot.herokuapp.com - I wasn't fully decided myself, but having thought about it a bit more, I think it would be better if you ran your own copy of Prout - fairly easy to do with a free Heroku account. Visit https://github.com/guardian/prout#run-your-own-instance-of-prout and hit the 'Deploy to Heroku' button.

—
Reply to this email directly or view it on GitHub.